doc/56_Search.md

```diff
@@ -74,14 +74,14 @@ Selecting a file will open it an an editor.
 
 ## Open Workspace Symbol
 
-Press <kbd>Ctrl+O</kbd> (<kbd>⌘+O</kbd> on macOS) to find all symbols in the workspace. The meaning
-of a symbol and the available symbol kinds are language-specific.
+Press <kbd>Ctrl+O</kbd> (<kbd>⌘+O</kbd> on macOS) to find symbols in the workspace. The meaning
+of a symbol and what symbols are available is language specific.
```

Comment:
sure no dash in language-specific?

Reply:
it should be there, adding to the follow-up